scijava/scijava-ops/src/main/java/org/scijava/ops/AnnotatedOpDependencyMember.java

```diff
@@ -74,6 +74,12 @@ public String getKey() {
 		return key;
 	}
 
+	@Override
+	public String getDescription() {
+		// TODO: consider more implementation here
```

Comment:
What sort of implementation? This would be for describing an op dependency, right? Probably we should allow that, indeed. Someone might want to document the purpose of each particular op dependency. Or do I misunderstand?

Reply:
@ctrueden right, the question is: In what ways should the user be able to declare the description? Should they use some field within the `@OpDependency` annotation, or should we use javadoc? If we use javadoc, should we use a particular tag, support some custom taglet for it, or take the entire javadoc? I'm leaning towards taking the entire javadoc, as it would be the simplest option...

Reply:
Okay, so I made a fix in 67ef0552. I decided that for an `OpDependency` in a method, we can just use the `@param` tag to describe it. For an `OpDependency` as a `Field`, we can just take whatever is in the bulk javadoc.

Unfortunately, however, this seems to lengthen my incubator build by about 30 seconds. This inefficiency should be addressed, likely via lazily viewing the javadoc...